apps/backend/src/mcp/server.py

```
@@ -1,470 +1,774 @@
-from apps.backend.src.mcp.tools.system_monitoring import SYSTEM_MONITORING_TOOLS
+# Load environment variables from .env file
+load_dotenv()
 
-# Import database initialization
-from apps.backend.src.core.database import init_database, close_database
+# Add the project root to Python path
+project_root = os.path.dirname(os.path.dirname(os.path.dirname(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))))
+sys.path.insert(0, project_root)
 
+# Local imports
+from apps.backend.src.core.database import init_database
```

Ruff (0.12.2)

Module level import not at top of file

(E402)